src/main/resources/view/JobListCard.fxml

```diff
@@ -7,18 +7,29 @@
 
 <!--Todo: update person-card style to card or separate into 2 kinds-->
 
-<HBox alignment="CENTER_LEFT" prefWidth="475.0" spacing="10.0" styleClass="person-card" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<HBox alignment="CENTER_LEFT" prefWidth="475.0" spacing="10.0" styleClass="person-card" xmlns="http://javafx.com/javafx/16" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
JavaFX 16

Perhaps this may help: https://stackoverflow.com/questions/65987397/how-to-set-preferred-version-in-scene-builder